frontend/integration-tests/tests/crud.scenario.ts

```diff
@@ -271,7 +271,7 @@ describe('Kubernetes resource CRUD operations', () => {
     it('displays YAML editor for creating a new custom resource instance', async() => {
       await browser.get(`${appHost}/k8s/cluster/customresourcedefinitions?name=${name}`);
       await crudView.isLoaded();
-      await crudView.resourceRows.first().element(by.linkText(crd.spec.names.kind)).click();
+      await crudView.resourceRows.first().element(by.linkText(crd.metadata.name)).click();
```

Comment:
I missed this change in the table. I think it would be better to keep the current behavior if possible since we already show the group in another column. Otherwise it's repeated.

Reply:
Sure thing, I'll change it back.